modules/custom-tuning-specification.adoc

```diff
@@ -148,9 +147,7 @@ ifdef::rosa-hcp-tuning[]
 ----
 <1> A TuneD profile to apply on a match. For example `tuned_profile_1`.
 <2> Profile ordering priority. Lower numbers mean higher priority (`0` is the highest priority).
-<3> Optional: A dictionary of key-value pairs `MachineConfig` labels. The keys must be unique.
-<4> If omitted, profile match is assumed unless a profile with a higher priority matches first or `machineConfigLabels` is set.
-<5> The label for the profile matched items.
+<3> The label for the profile matched items.
```

Comment:
I think we can keep the description of the <match> section above and reword this to just remove the "or `machineConfigLabels` is set." part.

```suggestion
<3> If omitted, profile match is assumed unless a profile with a higher priority matches first
<4> The label for the profile matched items.
```